speccurses.py

```diff
@@ -585,14 +709,13 @@ def _draw(stdscr, py, pad, index):
 
         k = stdscr.getch()
 
-#Utilities
+
+# Utilities
```

Comment:
We should split out this file into multiple smaller files if there are subsections in it.